### ISSUE TITLE
Question about the training batch size

### ISSUE THREAD

**Issue by KhoiDOO** [gaochao-s/Mesh-Silksong#5]
Hi!

I have a quick question about the training batch size. I saw that in [train_silksong_scratch_gpu16.sh](https://github.com/gaochao-s/Mesh-Silksong/blob/main/scripts/train_silksong_scratch_gpu16.sh), you leave the batch size as 2. From what I understand, the effective batch size will be 2 * 8 (8 GPUs from 2 nodes) = 16. Can you explain why you use a small batch size instead of 32, 64, or even 512?

Thank you!

**Comment by gaochao-s** [comment]
I only used batch size 2/4 for training, I did not try larger batch size. I remember the training loss can roughly reflect the training results, so I think: as long as your training loss is low enough (training loss < 0.25, eval loss doesn't increase) you will be able to achieve relatively good results.

**Comment by gaochao-s** [comment]
Based on my experience, you'd better set it to 1e-4. Remember do not set it to 1e-5 like EdgeRunner, it is too slow.
